Servers/UI/OJS.Servers.Ui/ClientApp/src/common/urls/compose-client-urls.ts

```diff
@@ -0,0 +1,30 @@
+import { CONTESTS_PATH } from './client-urls';
+
+const composeParticipationTypeResultsFullRoute =
+    (id: number, participationType: string) => `/${CONTESTS_PATH}/${id}/${participationType}/results/full`;
```

Comment:
You have a function for results defined below, why not reuse it?

Reply:
I have moved it to this file and deleted the old declaration. The idea behind that is so we can have the path declarations in a seperate file for the client-urls and the administraiton-urls, because in the old urls file there are path related to useHttp and the old architecture, and so in time we can move all the compositions of the urls inside: compose-cleint-urls and compose-administration-urls respectively

Reply:
Good approach, but I mean in the file itself. You have 2 functions that could be 1 or one can use the other..

Reply:
ah yes, didnt notice that, mb, its fixed